docs/eventing/samples/sinkbinding/README.md

```diff
@@ -0,0 +1,216 @@
+---
+title: "Sink Binding Example"
+linkTitle: "Sink Binding"
+weight: 10
+type: "docs"
+---
+
+A SinkBinding is responsible for linking together "addressable" Kubernetes
+resources that may receive events (aka the event "sink") with Kubernetes
+resources that embed a PodSpec (as `spec.template.spec`) and want to produce
+events.
+
+The SinkBinding can be used to author new event sources using any of the
+familiar compute abstractions that Kubernetes makes available (e.g. Deployment,
+Job, DaemonSet, StatefulSet), or Knative abstractions (e.g. Service,
+Configuration).
+
+
+## Create a CronJob that uses SinkBinding
+
+### Prerequisites
+
+1. Setup [Knative Serving](../../../serving).
+1. Setup [Knative Eventing and Sources](../../../eventing).
+
+### Prepare the heartbeats image
+
+Knative [event-sources](https://github.com/knative/eventing-contrib) has a
+sample of heartbeats event source. You could clone the source codes by
+
+```
+git clone -b "{{< branch >}}" https://github.com/knative/eventing-contrib.git
+```
+
+And then build a heartbeats image and publish to your image repo with
+
+```
+ko publish knative.dev/eventing-contrib/cmd/heartbeats
+```
+
+**Note**: `ko publish` requires:
+
+- [`KO_DOCKER_REPO`](https://github.com/knative/serving/blob/master/DEVELOPMENT.md#environment-setup)
+  to be set. (e.g. `gcr.io/[gcloud-project]` or `docker.io/<username>`)
+- you to be authenticated with your `KO_DOCKER_REPO`
+
+### Creating our event sink
+
+In order to verify our `SinkBinding` is working, we will create an Event Display
+Service that dumps incoming messages to its log.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
```

Comment:
```suggestion
        - image: gcr.io/knative-releases/github.com/knative/eventing-contrib/cmd/event_display
```

Reply:
Pretty sure I copied this from the container source one, FYI